packages/authgear-capacitor/android/src/main/java/com/authgear/capacitor/WebKitWebViewActivity.java

```diff
@@ -151,7 +151,7 @@ public boolean onCreateWindow(WebView view, boolean isDialog, boolean isUserGest
             }
             WebView.HitTestResult result = view.getHitTestResult();
             switch (result.getType()) {
-                case SRC_IMAGE_ANCHOR_TYPE: {
+                case SRC_IMAGE_ANCHOR_TYPE -> {
```

Comment:
Switch expression is a Java 17 features. Can we really use it? I prefer we stick with the old-fashioned switch statement.

Reply:
Updated to use old syntax